README.md

```diff
@@ -67,6 +67,20 @@ It's also useful to have an [alias](https://github.com/vchirikov/dotfiles/blob/7
 
 There are some shortcuts in `*.cmd` files, you can use them too.  
 
+## Access on https://local.actual.chat
+- Add line with ```local.actual.host``` aliases to [Hosts file](https://www.howtogeek.com/howto/27350/beginner-geek-how-to-edit-your-hosts-file/).
+ ```
+ 127.0.0.1  local.actual.chat media.local.actual.chat cdn.local.actual.chat
+ ```
+ - Import certificate [.\data\ssl\local.actual.chat.crt](./data/ssl/local.actual.chat.crt) to "Trusted Root Certification Authorities". You can do it with [Microsoft Management Console](https://www.thesslstore.com/knowledgebase/ssl-install/how-to-import-intermediate-root-certificates-using-mmc/#import-root-certificate-using-mmc12/) or [Chrome](https://www.pico.net/kb/how-do-you-get-chrome-to-accept-a-self-signed-certificate/).
+ - Run reverse proxy and image proxy with command.
+ ```powershell
+ docker-compose -f docker-compose.proxy.yml up -d
+ ```
+ - Run Actual-chat app.
+ - Navigate with browser to https://local.actual.chat/
```

Comment:
Signin redirects doesn't work now - redirect url is wrong = `http://actual-chat-app/signin-google`
we should pass env variable (public address) to the app

Reply:
fixed